bases/docusaurus.json

```diff
@@ -8,7 +8,8 @@
     "esModuleInterop": true,
     "jsx": "react",
     "lib": ["DOM"],
-    "moduleResolution": "Node16",
+    "module": "esnext",
+    "moduleResolution": "bundler",
```

Comment:
Related: #196 and https://github.com/facebook/docusaurus/pull/9050#discussion_r1236009465